interfaces/builtin/time_control.go

```diff
@@ -93,6 +93,11 @@ capability sys_time,
 /sys/class/rtc/*/ rw,
 /sys/class/rtc/*/** rw,
 
+# Allow access to pps
+# https://www.kernel.org/doc/html/latest/driver-api/pps.html
```

Comment:
Hmm it seems like pps can be used for other kinds of device not just time things, not sure if it makes sense here or not

Reply:
After some consideration I think this is fine to include here as the main usage of PPS is indeed for just setting the system clock time, and that's what it is being used for, so I think it's okay to include this here. Others may have differing opinions however

Reply:
All the practical usages of pps I've encountered were actually synchronizing with a reliable time source, usually supplied by GPS. I guess logically, you could feed that to ntpd to that fits into the time-control interface, but that's not necessarily the only consumer. OTOH, we don't have an interface for observing time, so time-control sounds like the only choice?